server/schema/user/user.model.js

```diff
@@ -79,9 +79,8 @@ const UserSchema = new Schema(
     },
     interestedTopics: [
       {
-        type: Number,
+        type: String,
```

Comment:
The topics we add are all categories and subcategories, right? Why change from number to string then?